elements/pfe-spinner/BaseSpinner.scss

```diff
@@ -0,0 +1,80 @@
+:host {
+  --pf-c-spinner--diameter: var(--pf-global--icon--FontSize--xl, 3.375rem);
```

Comment:
These should be defaults, instead of setting them on the host. Currently, users setting these properties higher up the tree will be surprised to find that their settings are overridden at the host element.

For those which are intentionally meant to override user configuration (perhaps e.g. `--pf-c-spinner__path--StrokeWidth`), those should be set on the shadow container